pkg/test/provisioner.go

```diff
@@ -66,7 +66,7 @@ func Provisioner(overrides ...ProvisionerOptions) *v1alpha5.Provisioner {
 	lo.Must0(raw.UnmarshalJSON(lo.Must(json.Marshal(options.Provider))))
 
 	provisioner := &v1alpha5.Provisioner{
-		ObjectMeta: ObjectMeta(options.ObjectMeta),
+		ObjectMeta: NamespacedObjectMeta(options.ObjectMeta),
```

Comment:
Technically this is a global resource, and should be ObjectMeta. The namespace is dropped by global CRDs, so it has no effect, but it's incorrect.